priv/test_files/unused_ignored_function_params/warnings_B.erl

```diff
@@ -0,0 +1,11 @@
+-module(warnings_B).
+
+-export([underscore/3, not_underscored/2]).
+
+%% _ is unused
+underscore(_, Arg2, Arg3) ->
+  {Arg2, Arg3}.
+
+%% Arg2 is unused and not underscored (although the linter should complain)
```

Comment:
Hank should not complain about `Arg2`, I would say.

Reply:
I think it doesn't complain. In that case, maybe we should more clearly state that in the comment.